### PR TITLE
Create testing framework for CLI

### PR DESCRIPTION
**Pull request by MGibson1**
## Type of change

- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other

## Objective
Add a testing framework for CLI.

The expected dev cycle would be to have two watchers, one for jslib and one for CLI tests. We could add jslib tests to this jasmine config, but it feels wrong to test a submodule.

## Code changes
* **package.json**: Add testing packages
* **helpers.ts**: Change console reporter (matches jslib reporter)
* **jasmine.json**: Specify test location to jasmine
* **tsconfig**: Jasmine requires `commonjs` modules and the `spec` directory to be included in transpilation.

## Testing requirements

None required

## Before you submit

- [x] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

**Comment by MGibson1** [comment]
Had to add a placeholder test since Jasmine returns an error if zero tests are found

File: package.json [hunk 1]
```diff
@@ -42,7 +42,9 @@
     "lint": "tslint 'src/**/*.ts' 'spec/**/*.ts' && prettier --check .",
     "lint:fix": "tslint 'src/**/*.ts' 'spec/**/*.ts' --fix",
     "prettier": "prettier --write .",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "test": "jasmine-ts -r tsconfig-paths/register -P spec/tsconfig.json",
+    "test:watch": "nodemon -w ./dist --delay 500ms --exec jasmine-ts -r tsconfig-paths/register -P spec/tsconfig.json"
```
Comment (Hinton) [inline package.json:47]: Any particular reason for the delay?

**Review by Hinton — APPROVED** [review]
As mentioned in slack, I believe we could use npm workspaces to "watch" multiple projects concurrently.

File: package.json (hunk 1 above)
Comment (MGibson1) [reply package.json:47]: I just copied it from jslib. I don't think we'll need any delay unless someone has problems with it continually restarting.

We shouldn't be watching dist, though.

**Review by Hinton — APPROVED** [review]
